### DISCUSSION TITLE
Query: Triple-Monitor Server with a client either side of those three

### DISCUSSION THREAD

**Discussion by ga2k** [debauchee/barrier#1333] [Q&A]
Just want to know if this is possible:

I have a triple-monitor setup as the server (Windows), and I want my Linux client to the left of the server's left monitor, and my macOS client to the right of the server's right monitor, like this...

[ Linux ] .. [ Windows # 1 ] .. [ Windows # 2 ] .. [ Windows # 3 ] .. [ macOS ]

Linux and Windows # 1 share a monitor, switching inputs via button on monitor,
macOS and Windows # 3 share a monitor, switching inputs via button on monitor

So when I want to work with Linux, Windows and Mac at the same time, I restrict myself to Windows # 2, switch Linux and macOS in using the buttons on my monitors, and Bob's your uncle. I would (of course) have a lot of "dead" movement between leaving the right hand side of Linux and arriving at Windows # 2, and again leaving Windows # 2 and arriving into macOS, but as I would only run in this mode occasionally, I don't mind. And if it annoyed me too much, I could always disconnect monitors 1 and  3 in Windows Display Settings.

Is this possible?

**Comment by eldahl** [comment]
yes.
Mouse movement and mouse focus on non visible screen areas will be troublesome though.

**Comment by ga2k** [comment]
Good to know it's possible. Are there any tutorials or videos showing how? Or do I just tell the Configuration that there are three PC's in the order I want (Linux - Windows - macOS) and it'll all just "work"?

**Comment by eldahl** [comment]
yes, simply configure and setup the server to your liking, then connect clients.

**Comment by arkag** [comment]
A similar, but I think more complicated question:

I have a single screen host and a triple screen client that I'm using. Is there a way to make it such that host edges go to different monitors?

|                           |                          |                           |
| ------------------------- | ------------------------ | ------------------------- |
| Client External Monitor 1 | Windows Desktop (Server) | Client External Monitor 2 |
|                           | Windows Laptop (Client)  |                           |

**Reply by xantrk** [reply]
I made a quick python script to monitor to log file for "switching to X"  messages, and when that happens triggers monitor to switch to HDMI or DP depending on that (using https://newam.github.io/monitorcontrol/ library)

**Comment by shymega** [comment]
Moving to discussion.
